packages/components/popover/src/PopoverContent/PopoverContent.styles.ts

```diff
@@ -9,5 +9,9 @@ export const getPopoverContentStyles = (isOpen: boolean) => ({
     borderRadius: tokens.borderRadiusMedium,
     boxShadow: tokens.boxShadowDefault,
     zIndex: tokens.zIndexDropdown,
+    '&:focus': {
+      outline: 'none',
+      boxShadow: tokens.glowPrimary,
```

Comment:
I remember @domarku didn't want that shadow around dropdown. Could you double check with him, please?

Reply:
hey @domarku the popover content had the browser default outline, here is how it look before and after:

**Before**
<img width="370" alt="Screenshot 2021-11-05 at 15 20 39" src="https://user-images.githubusercontent.com/6163988/140525575-9ef8a5da-6f0d-431b-a951-8f06568bfeb0.png">

**After**
<img width="299" alt="Screenshot 2021-11-05 at 15 20 10" src="https://user-images.githubusercontent.com/6163988/140525570-1ab65911-ca3d-4e7b-89ec-589f4392ddf0.png">

Reply:
Oh yeah, that makes sense! Let's use our styling. Just want to make sure it's for keyboard focus only, not when using the mouse to open the popover / menu.

Reply:
at the moment it's for both mouse and keyboard on popover, menu is a different behaviour as it focuses the first element inside the menu

Reply:
maybe we can use here `:focus-visible`, wdyt @mshaaban0 ?

Reply:
yeah nice, that'll work 👍🏼